controller/space_areas_test.go

```diff
@@ -154,14 +74,47 @@ func searchInAreaSlice(searchKey uuid.UUID, areaList *app.AreaList) *app.Area {
 	return nil
 }
 
-func createSpaceArea(name string, desc *string) *app.CreateSpaceAreasPayload {
+func (rest *TestSpaceAreaREST) TestListAreas() {
+	t := rest.T()
+	resource.Require(t, resource.Database)
 
-	return &app.CreateSpaceAreasPayload{
-		Data: &app.Area{
-			Type: area.APIStringTypeAreas,
-			Attributes: &app.AreaAttributes{
-				Name: &name,
-			},
-		},
+	/*
+		Space X --> Area 2 ---> Area 21-0
+	*/
+	var createdAreaUuids []uuid.UUID
+
+	parentArea := createSpaceAndArea(t, rest.db)
+	createdAreaUuids = append(createdAreaUuids, parentArea.ID)
+
+	parentID := parentArea.ID
+	name := "Area 2000A"
```

Comment:
Got you @hectorj2f , so you are suggesting I keep both the string I was using plus the name of the test. Sounds fine to me - however I am getting a `t.Name undefined (type *testing.T has no field or method Name)` - 
What am I missing? :/